srv/components/system/firewall/config.sls

```diff
@@ -30,8 +30,8 @@ include:
   firewalld.service:
     - name: {{ service }}
     - ports:
-      {% for port in salt['pillar.get']('firewall:' + nic + ':ports' + service ) %}
-      - port
+      {% for port in salt['pillar.get']('firewall:' + nic + ':ports:' + service ) %}
```

Comment:
`{% for port in salt['pillar.get']('firewall:' + nic + ':ports:' + service ) %}`
Improve this to:
`{% for port in pillar['firewall'][nic]['ports'][service] %}`